python/lsst/ts/ofc/scripts/generate_normalization_weights.py

```diff
@@ -0,0 +1,193 @@
+) -> np.ndarray:
+    """Compute the normalization weights.
+
+    Parameters
+    ----------
+    fwhm_matrix : np.ndarray
+        FWHM matrix.
+
+    Returns
+    -------
+    np.ndarray
+        Normalization weights.
+    """
+    m1m3_bending_range = ofc_data.m1m3_force_range / 20
+    m2_bending_range = ofc_data.m2_force_range / 20
+
+    m1m3_bmf = BendModeToForce("M1M3", ofc_data)
+    m2_bmf = BendModeToForce("M2", ofc_data)
+
+    range_weights = np.concatenate(
+        (
+            ofc_data.rb_stroke,
+            m1m3_bending_range / np.max(np.abs(m1m3_bmf.rot_mat), axis=0),
+            m2_bending_range / np.max(np.abs(m2_bmf.rot_mat), axis=0),
+        )
+    )
+
+    fwhm_weights = np.zeros(50)
+    for i in range(50):
+        fwhm_weights[i] = np.sqrt(np.sum(np.square(fwhm_matrix[:, i])))
+
+    normalization_weights = range_weights * fwhm_weights
+
+    return normalization_weights
+
+
+def save_normalization_weights(
+    normalization_weights: np.ndarray, output_dir: str
+) -> None:
+    """Save the normalization weights.
+
+    Parameters
+    ----------
+    normalization_weights : np.ndarray
+        Normalization weights.
+    output_dir : str
+        Output directory to save the normalization weights.
+    """
+    # Complete path to the yaml file
+    yaml_file_path = f"{output_dir}/normalization_weights.yaml"
+
+    # Write header and normalization weights to the yaml file
+    with open(yaml_file_path, "w") as yaml_file:
+        header = (
+            "---\n"
+            "# Normalization weights for the degrees of freedom\n"
+            "# Description: Normalization weights for the sensitivity matrix\n"
+            "#\n"
+            "# The normalization weights are used to normalize\n"
+            "# the sensitivity matrix and be able to operate in\n"
+            "# a different basis. The normalization weights are\n"
+            "# effectively a rescaling diagonal matix that is\n"
+            "# applied to the sensitivity to equalize the relevance\n"
+            "# of the different degrees of freedom (namelly, the\n"
+            "# different units and ranges of the degrees of freedom).\n"
+            "# The default is simply an identity (weights = 1)),\n"
+            "# which means that the sensitivity matrix is not\n"
+            "# normalized in any way. Units can be dimensionless.\n"
+            "# The reduced basis weights units are um of range * arcsec of FWHM.\n"
+            "# But the matrix is denormalized before retrieving the optical state,\n"
+            "# so it should not be relevant.\n\n"
+        )
+        yaml_file.write(header)
+        yaml.dump(normalization_weights.tolist(), yaml_file)
+
+
+def main(args: argparse.Namespace) -> None:
+    """Main function to generate normalization weights.
+
+    Parameters
+    ----------
+    args : argparse.Namespace
+        Arguments parsed from the command-line.
+    """
+    # Load the OFC data
```

Comment:
I think it is pretty obvious that you are loading OFCData.

Please, consider removing all the unnecessary comments. Comments should be used when what you are doing requires an explanation.